### PR TITLE
Add option to turn console view on/off

### PR DESCRIPTION
**Pull request by punx120**
THE FOLLOWING DISCLAIMER APPLIES TO ALL SOFTWARE CODE AND OTHER MATERIALS CONTRIBUTED IN CONNECTION WITH THIS SOFTWARE:
THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS “AS IS” AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.
ONLY THE SOFTWARE CODE AND OTHER MATERIALS CONTRIBUTED IN CONNECTION WITH THIS SOFTWARE, IF ANY, THAT ARE ATTACHED TO (OR OTHERWISE ACCOMPANY) THIS SUBMISSION (AND ORDINARY COURSE CONTRIBUTIONS OF FUTURES PATCHES THERETO) ARE TO BE CONSIDERED A CONTRIBUTION. NO OTHER SOFTWARE CODE OR MATERIALS ARE A CONTRIBUTION.

### CODE REVIEW

File: src/main/studio/kdb/Config.java [hunk 1]
```diff
@@ -439,11 +439,20 @@ public boolean isShowServerComboBox() {
         return Boolean.parseBoolean(p.getProperty("showServerComboBox","true"));
     }
 
+    public boolean isShowConsoleViewComboBox() {
```
Comment (gyorokpeter) [inline src/main/studio/kdb/Config.java:442]: Why do you include "ComboBox" in the name? It looks more like it is a button that is enabled by the setting.

**Review by gyorokpeter — CHANGES_REQUESTED** [review]

File: src/main/studio/kdb/Config.java (hunk 1 above)
Comment (punx120) [reply src/main/studio/kdb/Config.java:442]: it was just to be consistent with the showServer part even though i agree the name doesn't make a ton of sense as it's actually a "checkbox". Maybe it used to be a combobox and function name didn't get updated....

Comment (gyorokpeter) [reply src/main/studio/kdb/Config.java:442]: The server one is actually a combo box (colloquially it's called a dropdown list). So in that case the name make sense.
You need to name the new variable to reflect what it affects. From my reading of the code it seems it's the console view itself. So just calling it "showConsoleView" would be better.

Comment (punx120) [reply src/main/studio/kdb/Config.java:442]: I get what you're saying - updated the PR

**Review by gyorokpeter — APPROVED** [review]

**Pull request merged**